### PR TITLE
update CHANGES.rst for 0.11.0

### PR DESCRIPTION
**Pull request by stscicrawford**
Inclusion of pull requests and updates to the change log in preparation for 0.11.0.

### CODE REVIEW

**Comment by sosey** [comment]
either seems fine, leaving empty sections gives a quick overview of where the work was done though

**Comment by jdavies-st** [comment]
Agree empty sections are fine.

It would be good to standardize the tense used for each entry.  For commits, present tense ought to be used.  Perhaps the same for CHANGES.rst?  I don't know what the standard is elsewhere.  Right now we have a mix of present and past tense.

**Comment by stscicrawford** [comment]
Okay, I've updated the tenses to all be past tense and left the sections empty.

File: CHANGES.rst [hunk 1]
```diff
@@ -23,7 +23,7 @@ assign_wcs
 - Added a wavelength correction for the effective velocity of JWST
   relative to the barycenter.                                  [#2359, #2406]
 
-- NRC_TSGRISM assigns source location to set pixel [#2286]
+- UPdated NRC_TSGRISM to assign source location to set pixel [#2286]
```
Comment (philhodge) [inline CHANGES.rst]: This is really minor, but the "P" should be lower case.

File: CHANGES.rst [hunk 2]
```diff
@@ -86,16 +86,16 @@ dark_current
...
+- Added a new info method, similar to the method in astropy fits [#2268]
+
 - The ``DataModel`` ``__hasattr__`` method has been replaced by ``hasattr``.
   The former created the attribute when it was accessed. [#2275]
 
 - Improved error messaging when loading fits files into data models. [#2298]
 
 - New warning message when opening a file without DATAMODL keyword. [#2248]
 
-- New info method, similar to the method in astropy fits [#2268]
```
Comment (philhodge) [inline CHANGES.rst]: Did you intend to delete this without replacing it?

Comment (stscicrawford) [reply CHANGES.rst]: It's still there, I just moved it to the top of the list

**Pull request merged**